server/migrations/20181023192004-create-user.js

```diff
@@ -42,7 +42,7 @@ export default {
     },
     isConfirmed: {
       type: Sequelize.BOOLEAN,
-      allowNull: false,
+      allowNull: true,
```

Comment:
yo, if you're modifying a column please create a new migration

Reply:
I am sorry about this. Going forward I will create a new migration for each modification